applications/launchpad/gui-react/src/styles/themes/light.ts

```diff
@@ -39,7 +39,7 @@ const lightTheme = {
     background: styles.colors.darkMode.modalBackgroundSecondary,
     backgroundSecondary: styles.colors.darkMode.modalBackground,
     backgroundImage: styles.colors.light.backgroundImage,
-    accent: styles.colors.tari.purple,
+    accent: styles.colors.secondary.onTextLight,
```

Comment:
We need another key here for the bottom 'border' of Tabs in ExpertView, ie. `accentSecondary`, because we still need the purple color:

![image](https://user-images.githubusercontent.com/11715931/166955006-840afd58-0a5a-4298-b18e-e43e449bdba3.png)

After changing `accent`, now it has green color:

![image](https://user-images.githubusercontent.com/11715931/166955149-c459380d-03fa-4ca0-af7e-6aca7a30bbc4.png)


The purple color is also used here:

![image](https://user-images.githubusercontent.com/11715931/166955297-a3dc07e9-213a-4f60-8b0c-a8f528317613.png)

[https://www.figma.com/file/adOeOi2kRKn8mKM8GsrwZh/Tari-Launchpad-(Altalogy-Copy)?node-id=429%3A27274](https://www.figma.com/file/adOeOi2kRKn8mKM8GsrwZh/Tari-Launchpad-(Altalogy-Copy)?node-id=429%3A27274)

Reply:
added `inverted.accentSecondary`
for the green thing